README.md

```diff
@@ -1,20 +1,13 @@
-# ListView
+# Ember Collection
 
 [![Circle CI](https://circleci.com/gh/emberjs/ember-collection.svg?style=shield)](https://circleci.com/gh/emberjs/ember-collection)
 
-An efficient incremental rendering list view for large lists.
+An efficient incremental rendering component for large lists.
```

Comment:
yeah I agree. I'd prefer to minimize usage of the word "list" unless we're talking about specific layouts you can choose.